.github/workflows/docs-pages.yaml

```diff
@@ -35,10 +37,32 @@ jobs:
         run: make -C docs multiversion
       - name: Build redirects
         run: make -C docs redirects
+
+      # Generate EDSL documentation
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
+          java-version: "18"
+      - name: Set up Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Set up Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Build Docs
+        run: ./gradlew publishDocs
+      - name: Copy Docs To Publish Dir
+        run: cp -r docs/_build/dirhtml publish-docs-tmp; cp -r docs/edsl-api-docs/. publish-docs-tmp
```

Comment:
Can you confirm what the contents of publish-docs-tmp look like after this step?

Reply:
![Screen Shot 2022-10-14 at 3 18 17 PM](https://user-images.githubusercontent.com/13282627/195952028-360a85d0-d821-4f07-942f-95dd0cd422af.png)